reana_commons/validation/schemas/reana_analysis_schema.json

```diff
@@ -108,6 +108,24 @@
               "$id": "/properties/workflow/properties/resources/properties/kerberos",
               "type": "boolean",
               "title": "Kerberos authentication for the whole workflow."
+            },
+            "dask": {
+              "$id": "/properties/workflow/properties/resources/properties/dask",
+              "type": "object",
+              "title": "Dask cluster to be used by the workflow",
+              "properties": {
+                "image": {
```

Comment:
Please introduce also help for the various properties such as `image`. (By means of `description` fields, see .)